lib/orangetheses/indexer.rb

```diff
@@ -91,6 +91,14 @@ def title(dc_elements)
       titles.empty? ? nil : titles.first.text
     end
 
+    def title_sort(dc_elements)
+      titles = dc_elements.select { |e| e.name == 'title' }
+      title = titles.empty? ? nil : titles.first.text
+      unless title.nil?
+        title.downcase.gsub(/\p{^Alnum}/, '').gsub(/^(?:a)|(?:an)|(?:the)/, '')
```

Comment:
Good catch. Updated. We need to handle non-alphanumerics at the start of the string. Striping the article is harder if, e.g. the string starts with a quote.